### DISCUSSION TITLE
Upload content - Youtube link upload isn't working

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Lern/Community#8] [Issues]
Hi @amit-tarento, @keshavprasadms

Hope you are doing well!..

**Site Environments Details** - SunbirdEd portal of version 4.4.1

**Issue** - We are unable to upload content like Youtube link upload content isn't working.

**API Details** - 

**Request URL**: host/action/asset/v3/validate?field=license
**Request Method**: POST
**Status Code**: 400 Bad Request

**Payload -** 

```
{
    "asset": {
        "provider": "youtube",
        "url": "https://www.youtube.com/watch?v=E7LwQgUJnYQ"
    }
}
```

**Response -**

```
{
    "resmsgid": "b0ceb3fe-88b1-443e-b075-d4f261865003",
    "msgid": null,
    "err": "ERR_YOUTUBE_LICENSE_VALIDATION",
    "status": "failed",
    "errmsg": "Please Provide Valid YouTube URL!"
} 
```

Can you please look into it and let us know the steps to resolve the same.

cc @coolbung, @Renuka-S, @vinod303, @pranavirabatti

**Comment by komalm** [comment]
Hi @amit-tarento, @keshavprasadms - Please help us.

**Reply by amitec006** [reply]
@komalm This is not a Sunbird Lern BB related issue. Please raise this in Sunbird [Knowlg](https://github.com/Sunbird-Knowlg/Community/discussions) discussion forums.

**Reply by komalm** [reply]
@amit-tarento Thank you for your reply. I have created a new on for the same https://github.com/Sunbird-Knowlg/Community/discussions/16

Can you please close or delete this one, because I am unable to close or delete the same.

**Comment by rhwarrier** [comment]
Locking this thread, given it has been moved to Knowlg.
